src/aics-image-viewer/components/App/index.tsx

```diff
@@ -462,7 +462,7 @@ const App: React.FC<AppProps> = (props) => {
 
```

Comment:
Can't comment there, but should the expression on L455 above become `fullUrl === currentLoadSpec?.url`?

Reply:
Thank you! I will fix that.